### DISCUSSION TITLE
Elsa v3: Facing various issues with Designer

### DISCUSSION THREAD

**Discussion by viraj-ldce** [elsa-workflows/elsa-core#4093] [Q&A]
Hi,

I have integrated **3.0.0-rc1** packages for Elsa Server, API and Designer. I was trying to setup workflow from the designer but was facing issues related to functionality, usability and user experience. Below are some issues I faced with designer.

1.  When I tried to create and save workflow, It didn't happen straight away. It wasn't getting displayed in the Definition List. After 3-4 attempts it got saved and displayed under Definition List.

2. On the designer it seems few buttons are not working as expected and some buttons are overlapping other elements in design (It may just on the small screens).
![image](https://github.com/elsa-workflows/elsa-core/assets/128515082/cb2df117-19a9-4c24-bfef-2d1c85967dbc)

3. I did not find button to save workflow as a draft, there is a button to publish though. This publish button worked for me only the first time. After editing further and clicking publish it's not displaying any success / failure message.

4. The are below the the activity flow area for configuring activity settings, seems to be very small. (Again this may be because of small screen). I almost missed it as it was almost hidden at bottom.
![image](https://github.com/elsa-workflows/elsa-core/assets/128515082/5e7c786b-c15e-42b5-a494-8472935903fd)

5. After creating workflow, going back to home and coming back to edit workflow, the activities I configured were not displayed in the designer. I tried clicking Zoom to fit button but it didn't do any help, Also tried with Auto-Layout options but it didn't do any help either.

6. For now I just tried with Write Line activities and faced these issues. I haven't tried other activities yet.

Regards,
Viraj

**Comment by sfmskywalker** [comment]
Hi @viraj-ldce - thank you fir sharing your feedback and findings!

I have tried to reproduce these issues, but wasn't able to do so. If possible, would you mind sharing with me your solution files (zipped) so I can take a closer look?
